src/modules/elements/organizer-form/element.js

```diff
@@ -93,29 +97,20 @@ class OrganizerForm extends Component {
 				console.warning( 'Invalid creation of organizer:', newPost );
 			}
 
-			this.props.addOrganizer( newPost );
+			this.props.addOrganizer( newPost.id, newPost );
```

Comment:
Did we change the function signature somewhere? I'm not seeing it in this PR.

Reply:
@paulmskim I don't know why, but it wasn't working at all before. I found out the function requires those arguments

Reply:
but I'll double check that this same component isn't used by the standard organizer block, maybe that's why

Reply:
yeah it works everywhere

Reply:
hah... well probably why it wasn't working at all